lib/twilio/device.ts

```diff
@@ -1882,7 +1885,13 @@ namespace Device {
     getUserMedia?: any;
 
     /**
-     * Log level.
+     * Sets the log level.
+     *
+     * Possible values include any of the following numbers:
+     * 0 = trace, 1 = debug, 2 = info, 3 = warn, 4 = error, 5 = silent
+     *
+     * Or any of the following strings:
+     * 'trace', 'debug', 'info', 'warn', 'error', 'silent'
```

Comment:
If I recall correctly, the same strings in all-caps are also valid in the `logLevel` library.

Reply:
I checked the types and only numbers and lower case strings are accepted
```ts
declare namespace log {
    /**
     * Log levels
     */
    interface LogLevel {
        TRACE: 0;
        DEBUG: 1;
        INFO: 2;
        WARN: 3;
        ERROR: 4;
        SILENT: 5;
    }

    /**
     * Possible log level numbers.
     */
    type LogLevelNumbers = LogLevel[keyof LogLevel];

    /**
     * Possible log level descriptors, may be string, lower or upper case, or number.
     */
    type LogLevelDesc = LogLevelNumbers
        | 'trace'
        | 'debug'
        | 'info'
        | 'warn'
        | 'error'
        | 'silent'
        | keyof LogLevel;
```

Reply:
`keyof LogLevel` would include the all-caps variants of the strings.

Reply:
Ah true! Let me update the doc